e2e/__tests__/__snapshots__/failures.test.ts.snap

```diff
@@ -245,13 +262,20 @@ FAIL __tests__/duringTests.test.js
       ],
     }
 
-      43 | });
-      44 | 
-    > 45 | test('returned promise rejection', () => Promise.reject(deepObject));
+      48 | });
+      49 | 
+    > 50 | test('returned promise rejection', () => Promise.reject(deepObject));
          | ^
-      46 | 
+      51 | 
+
+      at Object.test (__tests__/duringTests.test.js:50:1)
+`;
+
+exports[`not throwing Error objects 6`] = `
+FAIL __tests__/throwObjectWithStackProp.test.js
+  ● Test suite failed to run
 
-      at Object.test (__tests__/duringTests.test.js:45:1)
+    42
```

Comment:
can we get

```
    thrown: Object {
      "stack": 42,
    }
```

here as well?

Reply:
I think yes, I will look into this too.

Reply:
just missing this now and we can merge 🎉